infra/modules/dms-networking/main.tf

```diff
@@ -1,6 +1,6 @@
 locals {
-  our_target_cidr_block   = var.dms_target_cidr_block # our [Nava] cidr block, where the target database for the DMS is located
-  their_source_cidr_block = var.dms_source_cidr_block # their [MicroHealth] cidr block, where the origin database for the DMS is located
+  our_target_cidr_block   = var.dms_target_cidr_block        # our [Nava] cidr block, where the target database for the DMS is located
+  their_source_cidr_block = var.grants_gov_oracle_cidr_block # their [MicroHealth] cidr block, where the origin database for the DMS is located
```

Comment:
non-blcker: What else is this cidr going to be used for?

Reply:
@aplybeah I/O for the foreign data wrapper, which is what this PR is allowing.